src/main/java/Incy.java

```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
+
+public class Incy {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String input;
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+        String logo = " ____  _  _  ___  _  _ \n"
+                    + "(_  _)( \\( )/ __)( \\/ )\n"
+                    + " _)(_  )  (( (__  \\  / \n"
+                    + "(____)(_)\\_)\\___) (__) \n";
```

Comment:
It is good to specify access modifiers for the variables being declared, like `private`, `public` or `protected`